Makefile

```diff
@@ -131,11 +148,35 @@ lint-static-check:
 lint: lint-static-check
 	$(LINT) run --timeout 5m
 
+.PHONY: multimod-verify
+multimod-verify: | $(MULTIMOD)
+	@echo "Validating versions.yaml"
+	$(MULTIMOD) verify
+
+COREPATH ?= "../opentelemetry-collector-contrib"
```

Comment:
I think we would need to have a sync core for both the collector core and contrib.

Reply:
Yes. I agree with you at this point but mostly this one is a default value but prioritizing between collector core and contrib, I would say the contrib is more beneficial for the default value.

Reply:
We can't do multiple?

Reply:
No we can't. Combine both is not an option since testing it out and return error. Creating an additional flag would not be an option too since it is wasteful for dev only used 1 flag. What I think was adding commenting in the Makefile and also additional notes in RELEASE doc if we used multimod in the future

Reply:
It looks like you can specify the path to the other repo with a flag `other-repo-root`.
https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/fdc91fd5380079ee89ea9315029a5b2304851b45/multimod/cmd/sync.go#L74
Couldn't you just have sync core run twice with both paths?

Reply:
After rethinking it, make sense to me now. Add additional default core of ```opentelemetry-collector``` when ```sync-core```